boards/riscv32/arty_bm310/board.h

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
Remove board.h if its not needed.

Reply:
Uses in building console driver

Reply:
So lets just have the driver include <soc.h> directly.

Reply:
@galak Sorry I didn't notice new notify so I will fix this in new amending :)